index.js

```diff
@@ -1,109 +0,0 @@
-const readline = require("readline");
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  let whichGame = await ask( //stores which game the user wants to play
-    'Do you (human) want to pick a number to guess, or should I (computer) pick a number for you (human) to guess?\nIf you want to pick a number, enter "1". If you want the computer to pick a number, enter "2".\n'
-  );
-  if (whichGame == 1) { //if else to determine which game is executed
-    let guess = 0; //Section to create variables that will be reassigned in the function.
-    let yOrN = 0;
-    let hOrL = 0;
-    let max = 100;
-    let min = 1;
-    let howManyTries = 0;
-    let whichGame = 0;
-
-    start();
-    async function start() {
```

Comment:
Watch out for variable shadowing! Lines 10-12 and 25-26 are cases of that happening. You could rename the second one based on the version of the game that the user is playing